pyproject.toml

```diff
@@ -32,11 +32,38 @@ xlrd = "^1.2.0"
 xlwt = "^1.3.0"
 
 [tool.poetry.dev-dependencies]
-psycopg2 = "< 2.9"  # https://github.com/psycopg/psycopg2/issues/1293
+psycopg2-binary = "^2.9.1"
 funcsigs = "^1.0.2"
 flake8 = "^3.8.4"
 Pillow = "^9.3.0"
 codecov = "^2.1.11"
+ruff = "^0.0.171"
+black = "^22.10.0"
+colorama = "^0.4.6"
+isort = "^5.10.1"
+
+[tool.black]
+line-length = 119
+
+[tool.ruff]
+line-length = 120
+select = ["E", "F", "W"]
+ignore = ["E501", "F405"]
+fix = true
+exclude = ["./.tox/*", "./.venv/*", "./env/*", "*/migrations/*", "./build/*"]
+
+[tool.isort]
+multi_line_output = 3
+force_grid_wrap = 0
+line_length = 119
+include_trailing_comma = true
+combine_as_imports = true
+sections = ["FUTURE", "STDLIB", "THIRDPARTY", "DJANGO", "DJANGOTHIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
```

Comment:
let's match rapidpro here and remove DJANGOTHIRDPARTY which never really made sense anyway